ofctrl/ofSwitch.go

```diff
@@ -165,40 +149,43 @@ func (self *OFSwitch) IsReady() bool {
 }
 
 // Handle switch connected event
-func (self *OFSwitch) switchConnected() {
-	self.changeStatus(true)
-
-	// Send new feature request
-	self.Send(openflow13.NewFeaturesRequest())
```

Comment:
FeatureRequest is sent in the connection when Controller receives "Hello" message from Switch, so this request is not needed.